packages/cli/src/storage.ts

```diff
@@ -31,6 +31,15 @@ export class FileSystemStorage extends ClientStorage {
     this.writeFile(data);
   }
 
+  getObject<T>(key: string): T | undefined {
+    const str = this.getString(key);
+    return str ? (JSON.parse(str) as T) : undefined;
+  }
+
+  setObject<T>(key: string, value: T): void {
+    this.setString(key, value ? JSON.stringify(value) : undefined);
```

Comment:
@codyebberson I replicated the one in core/src/storage.ts. If we change this, would we need to change the other storage files as well? 

https://github.com/medplum/medplum/blob/main/packages/core/src/storage.ts

Reply:
Yeah... If we do this, then we need to add `removeString` and `removeObject` to the interface in core, and make a bunch of changes to `MedplumClient` and tests in `storage.test.ts`.

I don't feel strongly one way or the other.  This feels a little out of scope for the current PR.

Tangentially related, one thing I would support is to change the behavior of `setObject` to persist the object, not the stringified value.

In core, we have to stringify, because its using `localStorage`, which only supports string values.

In cli, where we know we're writing to a JSON file, keeping it as an object will make the JSON file easier to read / hand edit.

Reply:
The removeString/Object is out of scope for this PR. Also changing the stringifiedValue to the object caused errors in other tests. Can discuss this more offline or tomorrow as well

Reply:
I'm ok with that 👍